liquibase-standard/src/main/java/liquibase/command/core/CalculateChecksumCommandStep.java

```diff
@@ -47,36 +70,36 @@ public List<Class<?>> requiredDependencies() {
 
     @Override
     public String[][] defineCommandNames() {
-        return new String[][] { COMMAND_NAME };
+        return new String[][]{COMMAND_NAME};
     }
 
     @Override
     public void run(CommandResultsBuilder resultsBuilder) throws Exception {
         CommandScope commandScope = resultsBuilder.getCommandScope();
-        final String changeSetIdentifier = commandScope.getArgumentValue(CHANGESET_IDENTIFIER_ARG);
+        final String changeSetPath = commandScope.getArgumentValue(CHANGESET_PATH_ARG);
+        final String changeSetId = commandScope.getArgumentValue(CHANGESET_ID_ARG);
+        final String changeSetAuthor = commandScope.getArgumentValue(CHANGESET_AUTHOR_ARG);
         final String changeLogFile = commandScope.getArgumentValue(CHANGELOG_FILE_ARG).replace('\\', '/');
         final Database database = (Database) commandScope.getDependency(Database.class);
 
-        List<String> parts = validateAndExtractParts(changeSetIdentifier, changeLogFile);
-        Scope.getCurrentScope().getLog(getClass()).info(String.format("Calculating checksum for changeset %s", changeSetIdentifier));
+        Scope.getCurrentScope().getLog(getClass()).info(String.format("Calculating checksum for changeset %s", changeSetId));
```

Comment:
I think in this log message we should also include the other two arguments as well (path and author) since the three together compose the unique id of a changeset. Otherwise, there could be cases of having the same id multiple times in a changeset and this message can add a bit of confusion around.

Reply:
Added both arguments